doc/api_assets/style.css

```
@@ -821,7 +820,7 @@ kbd {
 .header {
   position: sticky;
   top: -1px;
-  z-index: 1;
+  z-index: 2;
```

nit: we can use a bigger number in case we need to use a value in the middle
```suggestion
  z-index: 10;
```

---

sure :slightly_smiling_face::+1: